Autopilot/AttitudeManager/AttitudeManager.c

```diff
@@ -515,7 +515,15 @@ void setKValues(int type,float* values){
     for(i=0; i<7; i++){
        setGain(Kchannel[i],type,values[i]);
     }
-};
+}
+
+void setGains(int* type, float* values){
+    int gains[3] = {GAIN_KD,GAIN_KP,GAIN_KI};
```

Comment:
I dont think this is worth a for loop. We'll only ever have 3 types of gains. You can just do: 
```
setGain(type, GAIN_KD, values[0]);
setGain(type, GAIN_KP, values[1]);
setGain(type, GAIN_KI, values[2]);
```
Just add a comment on the function stating that values has to be an array of size 3, with the first value being KD, then KP, then KI to make it clear.